p2p/transport/webrtc/udpmux/mux.go

```diff
@@ -0,0 +1,194 @@
+package udpmux
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strings"
+	"sync"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/pion/ice/v2"
+	"github.com/pion/stun"
+)
+
+const receiveMTU = 1500
+
+var _ ice.UDPMux = &udpMux{}
+
+var log = logging.Logger("webrtc-transport-mux")
+
+type ufragConnKey struct {
+	ufrag  string
+	isIPv6 bool
+}
+
+type udpMux struct {
```

Comment:
Can you add an explanation here what exactly the lookup logic is? Look up by address, then look up by ufrag, etc.

Reply:
added

Reply:
It's a very long paragraph though. Might help to make it a bit lighter to read by splitting (some) stuff up in bullet point (like) lists

Reply:
It's already better, the last paragraph can probably be still improved, e.g. what you think of the following: 

```
// When a new packet is received on the underlying net.PacketConn:
// - we first check the address map to see if there is a connection associated with the remote address.
//   - If found we forward the packet to the connection.
//   - If an associated connection is not found:
//     - we check to see if the packet is a STUN packet.
//     - We then fetch the ufrag of the remote from the STUN packet and
//       use it to check if there is a connection associated with the (ufrag, IP address family) pair.
//       - If found we add the association to the address map.
//       - If not found, it is a previously unseen IP address and the `unknownUfragCallback` callback is invoked.
```

Not sure if my break up is even correct as the paragraph is even more ambigious and complex then I had feared. But you get the gist of what I am trying to get at, no?